lang/cs/Org.Apache.REEF.Common/Protobuf/ReefProtocol/REEFMessageCodec.cs

```diff
@@ -17,12 +17,18 @@
  * under the License.
  */
 
+using Org.Apache.REEF.Tang.Annotations;
 using Org.Apache.REEF.Wake.Remote;
 
 namespace Org.Apache.REEF.Common.Protobuf.ReefProtocol
```

Comment:
Should you move codec classes under Org.Apache.REEF.Common.Codec? They are not protobuf.